code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -2010,9 +2010,9 @@
 	chem_flags = NONE
 	taste_description = "plastic"
 
-/datum/reagent/glitter
-	name = "Generic Glitter"
-	description = "If you can see this description, contact a coder."
+/datum/reagent/glitter/standard
```

Comment:
```suggestion
/datum/reagent/glitter
```
What you made is actually making it not-standard.
/datum/reagent/glitter has no definition now.